docs/core/run-time-config/compilation.md

```diff
@@ -8,11 +8,118 @@ ms.topic: reference
 
 ## Tiered compilation
 
-- Configures whether the JIT compiler uses [tiered compilation](../whats-new/dotnet-core-3-0.md#tiered-compilation).
+- Configures whether the just-in-time (JIT) compiler uses [tiered compilation](../whats-new/dotnet-core-3-0.md#tiered-compilation). Tiered compilation includes two distinct behaviors: [quick JIT](#quick-jit) and the use of [ReadyToRun](../whats-new/dotnet-core-3-0.md#readytorun-images) (pre-compiled) code.
```

Comment:
Also, are you going to port the ReadyToRun info from the whats new to here?

Reply:
I think this goes along with @mairaw Maira's comment about the fact that we need to document tiered compilation conceptually. So I think the R2R info would go into that doc. I'll create an issue for it. Some of the TC detail in this config reference could then be pulled out to that doc too.

Reply:
I'm just referring to the table structure for the config vs xml vs environment variable setting. It looks so nice in here compared to the whats new 😄

Reply:
Ah, yes we have issue https://github.com/dotnet/docs/issues/16781, which I'll address separately after this merges to master.